### DISCUSSION TITLE
Jellyserr new install not showing verison

### DISCUSSION THREAD

**Discussion by sunnyghat** [seerr-team/seerr#791] [General]
Hello, 

I have performed a new installation if Jellyseer on Fedora 40 (non docker version), using the steps given here:  https://github.com/Fallenbagel/jellyseerr

however on starting the service the version is displayed like this:

![image](https://github.com/Fallenbagel/jellyseerr/assets/10565695/eb1005f3-a9f6-4eb9-8841-79a25504e631)

Is there a way to get this to display correctly>

during the installation the following was reported so not sure if this could be the cause:

`CYPRESS_INSTALL_BINARY=0 yarn install --frozen-lockfile --network-timeout 1000000
! Corepack is about to download https://registry.yarnpkg.com/yarn/-/yarn-1.22.22.tgz
? Do you want to continue? [Y/n] y

! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager

yarn install v1.22.22
[1/4] Resolving packages...
[2/4] Fetching packages...
warning swr@2.0.4: The engine "pnpm" appears to be invalid.
[3/4] Linking dependencies...
warning " > connect-typeorm@1.1.4" has incorrect peer dependency "typeorm@^0.1.4 || ^0.2.5".
warning "react-select > @emotion/react > @emotion/babel-plugin@11.10.5" has unmet peer dependency "@babel/core@^7.0.0".
warning "react-spring > @react-spring/konva@9.7.1" has unmet peer dependency "konva@>=2.6".
warning "react-spring > @react-spring/konva@9.7.1" has unmet peer dependency "react-konva@^16.8.0 || ^16.8.7-0 || ^16.9.0-0 || ^16.10.1-0 || ^16.12.0-0 || ^16.13.0-0 || ^17.0.0-0 || ^17.0.1-0 || ^17.0.2-0 || ^18.0.0-0".
warning "react-spring > @react-spring/native@9.7.1" has unmet peer dependency "react-native@>=0.58".
warning "react-spring > @react-spring/three@9.7.1" has unmet peer dependency "@react-three/fiber@>=6.0".
warning "react-spring > @react-spring/three@9.7.1" has unmet peer dependency "three@>=0.126".
warning "react-spring > @react-spring/zdog@9.7.1" has unmet peer dependency "react-zdog@>=1.0".
warning "react-spring > @react-spring/zdog@9.7.1" has unmet peer dependency "zdog@>=1.0".
warning " > react-toast-notifications@2.5.1" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning " > react-toast-notifications@2.5.1" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning " > @babel/cli@7.21.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > semantic-release-docker-buildx@1.0.1" has incorrect peer dependency "semantic-release@>=11.0.0 <18.0.0".
warning Workspaces can only be enabled in private projects.
[4/4] Building fresh packages...
$ husky install
husky - Git hooks installed
Done in 10.41s.
`

**Comment by fallenbagel** [comment]
> I have performed a new installation if Jellyseer on Fedora 40 (non docker version)

This is why the version isn't showing

**Reply by sunnyghat** [reply]
So is there a fix for this?

**Reply by fallenbagel** [reply]
> So is there a fix for this? 

This is not an issue per se for you to fix ( Its how local setups are currently works)

But before that, which branch are you on? Make sure you're on main branch and if so then you're on the latest version.

You can also check settings > about. If it says you're on develop branch, that's not good
